### ISSUE TITLE
Crear interfaz gráfica

### ISSUE THREAD

**Issue by iorch** [CodeandoGuadalajara/scan-o-matic#1]
- [ ] Pantalla inicial
- [ ] Cámara
- [ ] Presentación de resultados

**Comment by ponentesincausa** [comment]
propuesta (pero en smartphone) https://app.moqups.com/ponentesincausa@gmail.com/evL7Bhdd59/view/page/a19e73636

Hay que ver qué rollo con el scanner no-smartphone.

**Comment by iorch** [comment]
Sí, de hecho esta es una tareota, faltaría partirla en subtareas. 
Nuestra idea fue escribir las cosas generales que debemos hacer, pero podríamos ir partiéndolas en subtareas.

**Comment by marcelarosalesj** [comment]
WebRTC para usar la cámara con javascript
http://stackoverflow.com/questions/9644612/controlling-webcam-via-html5/9646193#9646193

Chequen lo de getUserMedia()
http://www.html5rocks.com/en/tutorials/getusermedia/intro/
http://www.html5rocks.com/en/tutorials/webrtc/basics/

**Comment by Ciip1996** [comment]
No lo queria cerrar jaja no puedo subir el zip les dejo la liga
https://www.dropbox.com/s/438n9rb3jldg452/Plantilla.zip?dl=0

**Comment by RickChihu** [comment]
Ya subí el archivo modificado a la carpeta correspondiente :smile:
